apps/fishing-map/features/datasets/datasets.utils.ts

```diff
@@ -39,6 +40,18 @@ export type SupportedDatasetSchema =
   | 'target_species' // between camelCase or snake_case
   | 'license_category'
 
+type IncomptibleFilter = {
+  id: SupportedDatasetSchema
+  value: any
+  disabled: SupportedDatasetSchema[]
+}
+type IncomptibleFiltersDict = Record<string, IncomptibleFilter[]>
+const INCOMPATIBLE_FILTERS_DICT: IncomptibleFiltersDict = {
+  'public-presence-viirs-match-prototype:v20220112': [
+    { id: 'matched', value: false, disabled: ['source', 'flag', 'shiptype', 'geartype'] },
```

Comment:
Do we plan to store this config diirectly in the dataset at some point?

Reply:
That is a good point, we could move it to the dataset.configuration object so we make dynamic and not code dependant.
I'd leave as it is for now so it is a bit easier to understand and we can release it now, but something to keep in mind